BLSQ/iaso
# IA-2756: Make sure all org units in the setuper account have geolocation

Explain what problem this PR is resolving

Related JIRA tickets : [IA-2756](https://bluesquare.atlassian.net/browse/IA-2756)

## Self proofreading checklist


## Changes

 - Cleaned the gpkg file to remove org unit without geolocation;
 - Skipped all org unit without geolocation when import via setuper:

## How to test

From the root folder, open sqlite session:
`sqlite3`

Open the database with:

`.open /setuper/data/small_sample.gpkg`

Set the table mode to print the data:
```
 .headers on
 .mode table
```


Check first the org units with no geocode on each level:


    BEGIN;
      SELECT COUNT(*) as FOSA_WITH_NO_GEOLOCATION FROM "level-6-Formation sanitaire" WHERE geom is null or parent_ref in(SELECT ref FROM "level-5-Commune" WHERE geom is null);
      SELECT COUNT(*) as COMMUNE_WITH_NO_GEOLOCATION FROM "level-5-Commune" WHERE geom is null or parent_ref in(SELECT ref FROM "level-4-District" WHERE geom is null);
      SELECT COUNT(*) as DISTRICT_WITH_NO_GEOLOCATION FROM "level-4-District" WHERE geom is null or parent_ref in(SELECT ref FROM "level-3-Province" WHERE geom is null);
      SELECT COUNT(*) as PROVINCE_WITH_NO_GEOLOCATION FROM "level-3-Province" WHERE geom is null or parent_ref in(SELECT ref FROM "level-2-Regional" WHERE geom is null);
      SELECT COUNT(*) as REGION_WITH_NO_GEOLOCATION FROM  "level-2-Regional" WHERE geom is null or parent_ref in(SELECT ref FROM "level-1-National" WHERE geom is null);
      SELECT COUNT(*) as COUNTRY_WITH_NO_GEOLOCATION FROM "level-1-National" WHERE geom is null;
    COMMIT;


You should get something look like:
![Capture d’écran du 2024-04-17 10-54-57](https://github.com/BLSQ/iaso/assets/6383219/9edf40db-406f-4ca6-aac9-32ee0509f2a2)

Then, launch the deletion of org unit without geolocation with:


    BEGIN;                      
            DELETE FROM "level-6-Formation sanitaire" WHERE geom is null or parent_ref in(SELECT ref FROM "level-5-Commune" WHERE geom is null);
            DELETE FROM "level-5-Commune" WHERE geom is null or parent_ref in(SELECT ref FROM "level-4-District" WHERE geom is null);
            DELETE FROM "level-4-District" WHERE geom is null or parent_ref in(SELECT ref FROM "level-3-Province" WHERE geom is null);
            DELETE FROM "level-3-Province" WHERE geom is null or parent_ref in(SELECT ref FROM "level-2-Regional" WHERE geom is null);
            DELETE FROM  "level-2-Regional" WHERE geom is null or parent_ref in(SELECT ref FROM "level-1-National" WHERE geom is null);
            DELETE FROM "level-1-National" WHERE geom is null;
    COMMIT;



Launch setuper and check in the org unit list, the org unit without geolocation. You should get 0 org unit!


## Print screen / video

[Capture vidéo du 15-04-2024 14:50:00.webm](https://github.com/BLSQ/iaso/assets/6383219/976e61e4-6ace-48e2-8595-c51fc8c831f4)



[IA-2756]: https://bluesquare.atlassian.net/browse/IA-2756?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
